src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Expression.cs

```diff
@@ -20,6 +20,11 @@ public abstract partial class Expression
         internal const string ExpressionRequiresUnreferencedCode = "Creating Expressions requires unreferenced code because the members being referenced by the Expression may be trimmed.";
         internal const string PropertyFromAccessorRequiresUnreferencedCode = "The Property metadata or other accessor may be trimmed.";
         internal const string GenericMethodRequiresUnreferencedCode = "Calling a generic method cannot be statically analyzed. It's not possible to guarantee the availability of requirements of the generic method. This can be suppressed if the method is not generic.";
+        internal const string GenericMethodRequiresDynamicCode = "Calling a generic method requires dynamic code generation.";
```

Comment:
Do we want a similar message here as we do for unreferenced code?

```suggestion
        internal const string GenericMethodRequiresDynamicCode = "Calling a generic method requires dynamic code generation. This can be suppressed if the method is not generic.";
```